### DISCUSSION TITLE
Where is repo for cli documentation

### DISCUSSION THREAD

**Discussion by mfoitzik** [quasarframework/quasar#14297] [General - Components / Directives / etc] [answered]
I came across a spot in the client docs (specifically [https://quasar.dev/quasar-cli/installation](https://quasar.dev/quasar-cli/installation)) that I wanted to send a pull request for a change. When I click on the edit icon I get a not found. Can someone point me to the repo for this page? Thanks.

**Comment by yusufkandemir** [comment, marked as answer]
That link is dead. You probably have some problems with the cache. You can visit the link in another browser or in incognito mode to see for yourself. I would suggest cleaning the cache, then you can see the latest versions of each page.

The documentation lives under the `docs` folder of this repository:
https://github.com/quasarframework/quasar/tree/dev/docs

**Reply by mfoitzik** [reply]
Thank you
